oat-sa/extension-tao-mediamanager
# feature: display asset export errors

Ticket: https://oat-sa.atlassian.net/browse/ADF-961

## What's Changed
- Display Asset Export errors in Background tasks panel.

## TODO 
- [ ] Update composer with packages

## How to test
- Create a new Passage Asset.
- In Authoring Edit the Asset and embed and image.
- Delete the embedded image - so there will be a broken link in the Asset.
- Try to Export the Asset.
- You should be able to notice the errors in the  Background tasks panel.

![Screenshot_2023-09-29_11-03-44](https://github.com/oat-sa/extension-tao-mediamanager/assets/8711268/200baf14-b43a-47b2-b2df-4a41a983eb45)

https://github.com/oat-sa/extension-tao-mediamanager/assets/8711268/1b626fcf-a0c2-434f-98a3-a044813c3be3
